release/models/system/openconfig-system-logging.yang

```diff
@@ -426,6 +432,12 @@ module openconfig-system-logging {
         "Source IP address for packets to the log server";
     }
 
+    leaf network-instance {
```

Comment:
With the addition of `network-instance` it's possible that `host` as a key is not unique anymore.